### PR TITLE
feat: Add apparmor profile

### PR DESCRIPTION
**Pull request by crypto-universe**
Increase security.

### CODE REVIEW

**Comment by crypto-universe** [comment]
@kpp , please take a look.

File: dpkg/usr.bin.tox-node [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#include <tunables/global>
+
+/usr/bin/tox-node {
+  #include <abstractions/base>
+  #include <abstractions/nameservice>
+
+  /etc/tox-node/config.yml r,
+  /sys/fs/cgroup/cpu,cpuacct/cpu.cfs_quota_us r,
```
Comment (kurnevsky) [inline dpkg/usr.bin.tox-node:8]: Why is this needed?

File: dpkg/usr.bin.tox-node [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  /usr/bin/tox-node mr,
+  /var/lib/tox-node/keys r,
+  owner /proc/*/cgroup r,
+  owner /proc/*/mountinfo r,
```
Comment (kurnevsky) [inline dpkg/usr.bin.tox-node:12]: And this one?

File: dpkg/usr.bin.tox-node (hunk 1 above)
Comment (crypto-universe) [reply dpkg/usr.bin.tox-node:8]: I did profiling and this the result of correct working daemon. Not sure _why_ it requires this access. On the other hand read-only access to cgroups should be safe.

File: dpkg/usr.bin.tox-node (hunk 2 above)
Comment (crypto-universe) [reply dpkg/usr.bin.tox-node:12]: Here tox has access to its _own_ mountinfo. It has rights to do so. Due to reasons - I didn't check which particular functionality requires that.

**Review by kurnevsky — APPROVED** [review]
Looks good to me, though I don't know much about apparmor and how it's used in debian.
